collection/sites/pastebin.yml

```
@@ -0,0 +1,10 @@
+name: Pastebin
+host: pastebin.com
+example: https://pastebin.com/9jEf44nc
+extract: "#pastebin\\.com/(?!u/)(?:\\w+(?:\\.php\\?i=|/))?(?'id'\\w+)#"
+iframe:
+    scrolling: "yes"
+    width: "100%"
+    height: "300"
+    style: ["border:none;resize:vertical;width:100%"]
+    src: //pastebin.com/embed_iframe/{@id}?theme=dark
```

I'd rather add regex (like in above comment) to allow users decide what theme to load, instead of hardcoding dark theme (just because it can be not good readable in some styles, like dark on dark etc.).
So it could look like 
```suggestion
    src: //pastebin.com/embed_iframe/{@id}{@dark}
```

Then one can post either `https://pastebin.com/9jEf44nc` or `https://pastebin.com/9jEf44nc?theme=dark` to choose the dark theme option if needed.